client/data_definition.md

```diff
@@ -0,0 +1,65 @@
+# Data Definition in a DefraDB instance
+
+Data held in a DefraDB instance is organized into [collections](#collections) of documents.  [Collections](#collections) are [local](#local-definitions) groupings of documents that share the same [globally](#global-definitions) defined shape declared by a [schema](#schemas).
+
+## Local definitions
+
+Local definitions are specific to the node you are directly working with, they are not shared with, or assumed to be the same on other nodes in the network.
+
+Splitting local elements out from the global ones allows some local customization to the way data is organized within any given node.  It also minimizes the amount of 'stuff' that must be kept consistent across the decentralized network in order to have a well behaving database.
+
+Local data definitions are always defined on the [collection](#collections).
+
+Examples include indexes, field IDs, and [lens transforms](https://docs.source.network/defradb/guides/schema-migration).
+
+## Global definitions
+
+Global definitions are consistent across all nodes in the decentralized network of nodes in the database. This is enforced by the use of things like CIDs for schema versions.  If a global definition was to differ across nodes, the different variations will be treated as a completely different definitions.
+
+Global data definitions are always defined on the [schema](#schemas).
+
+Examples include field names, field kinds and [CRDTs](https://docs.source.network/defradb/guides/merkle-crdt).
+
+## Collections
+
+Collections represent [local](#local-definitions), independently queryable datasets sharing the same shape.
+
+Collections are defined by the `CollectionDescription` struct.  This can be mutated via the `PatchCollection` function.
+
+A collection will always have a [global](#global-definitions) shape defined by a single [schema](#schemas) version.
+
+### Versions
+
+`CollectionDescription` instances may be active or inactive.  Inactive `CollectionDescription`s will not have a name, and cannot be queried.
+
+When a new [schema](#schemas) version is created for a schema that has a collection defined for it, a new `CollectionDescription` instance will be created and linked to the new schema version.  The new `CollectionDescription` instance will share the same root ID as the previous, and may be active or inactive depending on what arguments the user defining the new schema specified.
```

Comment:
suggestion: When a new [schema](#schemas) version is created ~for a schema that~ *and* has a collection defined for it, a new `CollectionDescription` instance will be created and linked to the new schema version.

Reply:
Yeah, I think I agree - will do, thanks Keenan :)

- [ ] Remove extra words

Reply:
praise: The italics on `and` in your suggestion saved me a bit of hassle, thanks for bothering to add that :)